src/makeRobot.js

```diff
@@ -37,8 +37,77 @@
  *
  * @return {Robot}
  */
+
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    get info() {
+      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
+    },
+
+    get location() {
+      return `${name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    goForward(value) {
+      if (value > 0) {
+        this.coords.y += value;
+      } else if (value < 0) {
+        this.coords.y += 0;
+      } else {
+        this.coords.y++;
+      }
+
+      return this;
+    },
```

Comment:
You can greatly simplify your code by just setting a default value. Please apply to other functions as well
```suggestion
    goForward(value = 1) {
      if (value > 0) {
        this.coords.y += value;
      } 
      
      return this;
    },
```